docs/only-allow-admins-to-create-new-streams-feature.md

```diff
@@ -0,0 +1,12 @@
+# Only allow admins to create new streams
+
+1. Login with an Admin account.
+2. Locate the cog
+(![cog](/static/images/help/cog.png)) on the main navigation bar and click on
```

Comment:
Add "icon" after the cog image.

Can you be more specific in your location? "Upper right corner of the right sidebar" might be more accurate